nservicebus/outbox/index.md

```diff
@@ -79,8 +79,60 @@ In order to gradually convert an entire system from the DTC to the Outbox:
 WARNING: When verifying outbox functionality, it can sometimes be helpful to temporarily [stop the MSDTC Service](https://technet.microsoft.com/en-us/library/cc770732.aspx). This ensures that the Outbox is working as expected, and no other resources are enlisting in distributed transactions.
 
 
+## Message identity and idempotent processing
+
+If handlers are idempotent then outbox might not be required at all. When implementating idempotent processing keep in mind that any generated message *must* apply the same message identifier if invoked more-then-once to have deduplication working on the infrastructure level or to solve this at the logical level by validating message data. [Read more on how to set a message identifier.](/nservicebus/messaging/message-identity.md)
+
+
+## Concurrency
+
+Outbox cannot prevent that handlers or sagas are invoked at most once. Under certain scenarios this can happen:
```

Comment:
how about:
Outbox cannot guarantee that handlers or sagas are invoked exactly once.

Reply:
> Under certain scenarios this can happen:

this is abit confusing as it gives the impression the following bullet points are a sequence of actions. how about:

> The following scenarios might cause handlers to be invoked multiple times: